src/results/electric_tariff.jl

```diff
@@ -34,42 +34,42 @@ Adds the ElectricTariff results to the dictionary passed back from `run_reopt` u
 Note: the node number is an empty string if evaluating a single `Site`.
 
 ElectricTariff results:
-- `lifecycle_energy_cost` lifecycle cost of energy from the grid in present value, after tax
-- `year_one_energy_cost` cost of energy from the grid over the first year
-- `lifecycle_demand_cost` lifecycle cost of power from the grid in present value, after tax
-- `year_one_demand_cost` cost of power from the grid over the first year
-- `lifecycle_fixed_cost` lifecycle fixed cost in present value, after tax
-- `year_one_fixed_cost` fixed cost over the first year
-- `lifecycle_min_charge_adder` lifecycle minimum charge in present value, after tax
-- `year_one_min_charge_adder` minimum charge over the first year
+- `lifecycle_energy_cost_after_tax` lifecycle cost of energy from the grid in present value, after tax
```

Comment:
lots of breaking changes with these name changes. please document in the CHANGELOG